.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: csharp
+sudo: true
+mono: none
+dist: xenial
+dotnet: 2.2.203
+os:
+  - linux
+  - osx
+branches:
+  only:
+    - master
+script:
+  - dotnet restore
+  - dotnet build -c Release
+  - dotnet test -c Release -v n
+#deploy:
+#  skip_cleanup: true
+#  provider: script
+#  script: dotnet nuget push ${TRAVIS_BUILD_DIR}/**/*.nupkg -k ${NUGET_API_KEY}
```

Comment:
This will perform automatic nuget publish when version is updated

Reply:
@JohannesMoersch I would suggest putting your encoded nuget api key here to enable automatic nuget publish